src/cycling/Team.java

```diff
@@ -19,19 +24,29 @@ public int[] getRiderIDArray() {
         for (Integer riderID : ridersHash.keySet()) {
             riderIDArray[position++] = riderID;
         }
-
+        
         return riderIDArray;
     }
 
-    
+    // Adds Rider object as a value in ridersHash with a key of its Rider's ID
+    public void addRider(Rider newRider) {
+
+        Integer riderID = newRider.getRiderID();
+
```

Comment:
Note here you are working with the wrapped Integer type instead of int, which might cause problems later